quickstart/deploy-a-test-application.md

```diff
@@ -50,12 +50,16 @@ If you used the storage class example from previous stage, then volume binding m
 
 ## Deploy the FIO Test Pod
 
-We schedule the application to one of the storage nodes in order to make the volume available to the application in spite of the node failures. That's why nodeSelector is used in the Pod specification. See ["local" storage class parameter description](https://mayastor.gitbook.io/introduction/reference/storage-class-parameters#local) for more in-depth explanation of how the scheduling works.
+The Mayastor CSI driver will cause the application pod and the corresponding Mayastor volume's NVMe target/controller ("Nexus") to be scheduled on the _same_ Mayastor Node, in order to assist with restoration of volume and application availabilty in the event of a storage node failure.
+
+{% hint style="warning" %}
+In this version, applications using PVs provisioned by Mayastor can only be successfully scheduled on Mayastor Nodes.  This behaviour is controlled by the `local:` parameter of the corresponding StorageClass, which by default is set to a value of `true`.  Therefore, this is the only supported value for this release - setting a non-local configuration may cause scheduling of the application pod to fail, as the PV cannot be mounted to a worker node other than a MSN.  This behaviour will change in a future release.
```

Comment:
Are there extra spaces after full-stops?

Reply:
https://www.cultofpedagogy.com/two-spaces-after-period/  :)

Reply:
Will adopt this style in future text.  At this point, double sentence spacing is already used throughout this page and the documentation in general.